tests/s3_test.go

```diff
@@ -0,0 +1,85 @@
+package tests
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/aws/aws-sdk-go/service/s3/s3manager"
+	"github.com/gruntwork-io/terratest/modules/aws"
+	"github.com/gruntwork-io/terratest/modules/logger"
+	"github.com/gruntwork-io/terratest/modules/random"
+	"github.com/gruntwork-io/terratest/modules/retry"
+	"github.com/gruntwork-io/terratest/modules/terraform"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestTerraformS3Bucket(t *testing.T) {
+	t.Parallel()
+
+	expectedName := fmt.Sprintf("terratest-aws-s3-example-%s", strings.ToLower(random.UniqueId()))
+
+	awsRegion := aws.GetRandomStableRegion(t, nil, nil)
+
+	terraformOptions := terraform.WithDefaultRetryableErrors(t, &terraform.Options{
+		TerraformDir: "../examples/test_minimal",
+
+		Vars: map[string]interface{}{
+			"test_bucket_name": expectedName,
+			"aws_region":       awsRegion,
+		},
+
+		EnvVars: map[string]string{
+			"AWS_DEFAULT_REGION": awsRegion,
+		},
+	})
+
+	defer terraform.Destroy(t, terraformOptions)
+
+	terraform.InitAndApply(t, terraformOptions)
+
+	b, err := terraform.OutputMapOfObjectsE(t, terraformOptions, "test-bucket")
+
+	if err != nil {
+		logger.Log(t, err)
```

Comment:
Yes, this is a mistake. We don't want to use `terraform.OutputMapOfObjectsE` here, and call `terraform.OutputMapOfObjects` instead which would not raise error but fail the test in case it had to raise one. (we leave the error handling to the Terratest lib in this case.).